<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Tests\App\Models\LigneLivraisonModelTest" file="C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\tests\LigneLivraisonModelTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.000000">
    <testcase name="testQtePositivePrixZero" file="C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\tests\LigneLivraisonModelTest.php" line="25" class="Tests\App\Models\LigneLivraisonModelTest" classname="Tests.App.Models.LigneLivraisonModelTest" assertions="0" time="0.000000">
      <error type="CodeIgniter\Database\Exceptions\DatabaseException">Tests\App\Models\LigneLivraisonModelTest::testQtePositivePrixZero&#13;
CodeIgniter\Database\Exceptions\DatabaseException: Table 'users' already exists
&#13;
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\BaseConnection.php:692
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\Forge.php:570
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\app\Database\Migrations\2025-04-20-203852_CreateUsersTable.php:27
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\MigrationRunner.php:866
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\MigrationRunner.php:182
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\DatabaseTestTrait.php:175
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\DatabaseTestTrait.php:120
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\DatabaseTestTrait.php:60
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\CIUnitTestCase.php:251
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\tests\LigneLivraisonModelTest.php:19</error>
    </testcase>
    <testcase name="testPrixNegatif" file="C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\tests\LigneLivraisonModelTest.php" line="41" class="Tests\App\Models\LigneLivraisonModelTest" classname="Tests.App.Models.LigneLivraisonModelTest" assertions="0" time="0.000000">
      <error type="CodeIgniter\Database\Exceptions\DatabaseException">Tests\App\Models\LigneLivraisonModelTest::testPrixNegatif&#13;
CodeIgniter\Database\Exceptions\DatabaseException: Table 'users' already exists
&#13;
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\BaseConnection.php:692
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\Forge.php:570
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\app\Database\Migrations\2025-04-20-203852_CreateUsersTable.php:27
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\MigrationRunner.php:866
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Database\MigrationRunner.php:182
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\DatabaseTestTrait.php:175
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\DatabaseTestTrait.php:120
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\DatabaseTestTrait.php:60
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\system\Test\CIUnitTestCase.php:251
C:\Users\ce pc\Desktop\devops atelier2\AuTestWin\tests\LigneLivraisonModelTest.php:19</error>
    </testcase>
  </testsuite>
</testsuites>
